pandas/io/gbq.py

```diff
@@ -46,8 +46,8 @@ def _test_google_api_imports():
 
     try:
         import httplib2  # noqa
-        from apiclient.discovery import build  # noqa
```

Comment:
I agree its better to fallback so that older version don't break. I've pushed a new version